homework_1/src/main/java/Homework1.java

```diff
@@ -0,0 +1,57 @@
+import java.util.Random;
+
+public class Homework1 {
+    public static void main(String[] args){
+        Homework1 instance = new Homework1();
+        int length =15;
+        int [] array = new int[length];
+        array = instance.getRandomArray(length);
+        instance.printArray(array);
+
+        System.out.print("\n");
+        System.out.println(instance.chet(array));
+        instance.printArray(instance.sortArray(array));
+    }
+    private int chet (int [] arr){
+        int c =0;
```

Comment:
c - not clear name let's switch to the counter or some similar

Reply:
Done.